code/modules/atmospherics/machinery/datum_pipeline.dm

```diff
@@ -129,6 +129,9 @@
 
 /obj/machinery/atmospherics/components/addMember(obj/machinery/atmospherics/A)
 	var/datum/pipeline/P = returnPipenet(A)
+	if(!P)
+		throw EXCEPTION("Called null.addMember in datum_pipeline.dm, line 132")
```

Comment:
Same as above.

Reply:
might want to add some more useful information to this crash than just.. the line number

Reply:
there's no need to put the line number in the first place, CRASH handles it